week-1-practise/level-1/9-function-output-2.js

```diff
@@ -7,8 +7,19 @@ console.log(result);
 
 // Answer these questions:
 // - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// the function concatenates two strings
+
 // - How many parameters does the function take?
+//   it takes two parameters
+
 // - What is the function name?
+// it is called s
+
 // - Where is the function called? with what parameters?
+// it is assigned to a variable called result and called with two strings ("code" and "your")
+
 // - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+//it returns string
+
+// - How can this function be improved?
+// we could use trime function and also we could print out the final stirng with upper case letters using toUppercase()
```

Comment:
There's a more obvious problem with this function as defined. Can you think of one? How do we know what the function does from its name `s`?